plugins/puppetdb/README.md

```diff
@@ -0,0 +1,63 @@
+# PuppetDB Plugin
```

Comment:
Can you add the screenshots? Sorry, I feel like I left a comment about this but it's gone now 🤔 

The screenshots really help to give people a feel for the features. You could create a Features section like the Azure DevOps plugin does to showcase them - https://github.com/backstage/backstage/tree/master/plugins/azure-devops

Reply:
Sorry, I think I was just expecting them at the top and didn't scroll down. Thanks!